client/my-sites/plans-features-main/index.jsx

```diff
@@ -428,6 +461,37 @@ export class PlansFeaturesMain extends Component {
 		return false;
 	}
 
+	renderSecondaryFormattedHeader() {
+		const { siteSlug, translate } = this.props;
+		let headerText;
+		let subHeaderText;
+		if ( this.isJetpackBackupAvailable() ) {
+			headerText = translate( 'Plans' );
+			subHeaderText = translate(
+				'Get everything your site needs, in one package — so you can focus on your business.'
+			);
+		}
+		if ( this.isDisplayingPlansNeededForFeature() ) {
+			headerText = translate( 'Upgrade plans to access this feature and more' );
```

Comment:
@obi2020 Do you have any suggestion for this copy? For context, we're going to show this when a user clicks on an upsell nudge advertising a feature that is included only on a few selected plans. We'll hide the plans not containing the upsold feature, but we'll show a link so the user can view all plans.

<img width="1177" alt="Screenshot 2020-04-16 at 11 41 56" src="https://user-images.githubusercontent.com/43215253/79447343-a0299f80-7fd7-11ea-93c6-4ede3dd311f3.png">